Calculator/Calculator/Model/Formula.swift

```diff
@@ -7,7 +7,28 @@
 
 import Foundation
 
-struct Formula<T> {
-    var operands: CalculatorItemQueue<Double> = CalculatorItemQueue()
-    var operators: CalculatorItemQueue<Operator> = CalculatorItemQueue()
+
+struct Formula {
+    var operands: CalculatorItemQueue<Double>
+    var operators: CalculatorItemQueue<Operator>
+    
+    public mutating func result() throws -> Double {
+        guard var result = operands.dequeue() else {
+            throw CalculateError.unavailableDequeue
+        }
+        guard operands.count == operators.count else {
+            throw CalculateError.operandOperatorCountNotMatching
+        }
+        for _ in 1...operators.count {
```

Comment:
```suggestion
        guard operands.count == operators.count else {
            throw CalculateError.operandOperatorCountNotMatching
        }
        
        for _ in 1...operators.count {
```
=> guard 와 for 문 사이에 띄워쓰기 한번 해주면 더 깔끔할 듯 합니다.